### PR TITLE
Fix accuracy calculation

### PR DESCRIPTION
**Pull request by BAILOOL**
The division of numerator and denominator returns an integer which equals to 0. Thus, the accuracy for any epoch is printed as 00.00%. Solution: Added casting of the numerator to float.

### CODE REVIEW

**Comment by moskomule** [comment]
Thank you for your PR!
Indeed the original works fine in my environment. Can you let me know your environment (Python/PyTorch versions etc..).

**Comment by BAILOOL** [comment]
Python: 2.7; PyTorch: 0.3.0.post4. Actually, it should be fine since [in here](http://pytorch.org/docs/master/torch.html) it is written ```torch.sum(input) → float```. Though for my settings it returns an integer.

**Comment by moskomule** [comment]
Ok
